assembly/src/main/dist/docker/build.sh

```diff
@@ -17,7 +17,7 @@ cd ../../../../..
 MVN_VERSION=$(xmllint --xpath "//*[local-name()='project']/*[local-name()='version']/text()" pom.xml)
 
 mvn clean
-mvn -T 2 -Passembly install -DskipTests -Dmaven.javadoc.skip=true -Dmaven.test.skip=true -Dformatter.skip=true -Dimpsort.skip=true -Dxml-format.skip=true  -Djapicmp.skip -Denforcer.skip=true -Dbuildnumber.plugin.phase=none -Danimal.sniffer.skip=true
+mvn -T 2 -Passembly install -DskipTests -Dmaven.javadoc.skip=true -Dformatter.skip=true -Dimpsort.skip=true -Dxml-format.skip=true  -Djapicmp.skip -Denforcer.skip=true -Dbuildnumber.plugin.phase=none -Danimal.sniffer.skip=true
```

Comment:
@jeenbroekstra remember you got an error the first time you ran this command, I belive it was because we were skipping everything in the test scope which probably broke those builds that depended on the test scope (like compliance?)